clang/Util.d

```diff
@@ -112,7 +122,35 @@ File namedTempFile(string prefix, string suffix)
     scope (exit) close(fd);
 
     if (fd == -1)
-        throw new NamedTempFileException("Cannot create \"%s\" temporary file.".format(path));
+    {
+        auto message = format("Cannot create temporary file \"%s\".", path);
```

Comment:
I'm not sure if by instance variable you mean "pass the result directly as the argument", but if is so it is fixed. I sometimes add such an auxiliary variables to shorten the line length and avoid unnecessary nesting in function argument list.